docs/services/motion/algorithms.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "About the Motion Planning Algorithms"
+linkTitle: "Algorithms"
+weight: 20
+type: "docs"
+description: "Information about the motion planning algorithms Viam uses."
+---
+
+Viam implements two planning algorithms, both based in principle on [RRT](https://en.wikipedia.org/wiki/Rapidly-exploring_random_tree):

```

Comment:
I think before diving into each algorithm, it would be good to explain here that Viam first starts with one and then uses the other (the last paragraph on this page currently). And then dive into each algorithm

Reply:
I added a heading down there, but I'm not sure how to move it up here when the concepts haven't been introduced yet.

Reply:
```suggestion
Robotic motion planning is heavily reliant on planning algorithms to determine how to achieve motion for a particular scenario.
Many algorithms already exist for this problem, and motion planning is a domain where improvements and novel developments occur frequently.
Viam does not implement all motion planning algorithms but has implemented two in its strategy for planning general robot motion.
The two algorithms Viam supports are both based in principle on [RRT](https://en.wikipedia.org/wiki/Rapidly-exploring_random_tree):
```